lib/promiscuous/publisher/operation/base.rb

```diff
@@ -101,15 +101,15 @@ def queue_operation_payloads(operations = self.operations)
   end
 
   def payloads
-    @operation_payloads.map do |_, operation_payloads|
+    @operation_payloads.map do |lock_key, operation_payloads|
       payload              = {}
       payload[:operations] = operation_payloads
       payload[:app]        = Promiscuous::Config.app
       payload[:timestamp]  = Time.now
       payload[:generation] = Promiscuous::Config.generation
       payload[:host]       = Socket.gethostname
+      payload[:key]        = lock_key
       payload.merge!(self.payload_attributes)
-      MultiJson.dump(payload)
```

Comment:
if there is a way to get at this `lock_key` from the `instance` we can put this back and refactor